src/EventTickets/Actions/RegisterEventsMenuItem.php

```diff
@@ -26,6 +29,18 @@ public function __invoke()
      */
     public function render()
     {
+        if(isset($_GET['id'])) {
```

Comment:
@pauloiankoski we can update the check to account for the re-use of the id parameter.

```suggestion
        if(isset($_GET['id']) && 'new' !== $_GET['id']) {
```